src/python/pants/backend/jvm/tasks/jvm_compile/rsc/rsc_compile.py

```diff
@@ -273,6 +275,17 @@ def _rsc_key_for_target(self, compile_target):
     else:
       raise TaskError('unexpected target for compiling with rsc .... {}'.format(compile_target))
 
+  def _metacp_dep_key_for_target(self, compile_target):
+    if self._only_zinc_compileable(compile_target):
```

Comment:
maybe pick one - compileable or compilable (i.e. _only_zinc_compileable vs _rsc_compilable)

Reply:
lol nice catch. I think I'll clean that up in a separate name focused change.